SuperBuild/External_elastix.cmake

```diff
@@ -26,12 +26,12 @@ if(NOT DEFINED ${proj}_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
   endif()
 
   message(STATUS "ITK version: ${ITK_VERSION_MAJOR}.${ITK_VERSION_MINOR}.${ITK_VERSION_PATCH}.")
-  if(DEFINED ITK_VERSION_MAJOR AND ${ITK_VERSION_MAJOR}.${ITK_VERSION_MINOR} VERSION_LESS 5.0)
-    set(ELASTIX_GIT_REPOSITORY "$https://github.com/SuperElastix/elastix.git")
```

Comment:
The dollar sign here (`"$https://.../elastix.git"`) looks unfamiliar to me. Was it a typo? It appears to be introduced on March 21, 2022, with commit 3e9ef490ee6a6f901daa3280af112950f6af13ec

If the dollar sign would have caused an error when retrieving elastix.git, I guess ITK < 5.0 was "de facto" already dropped in 2022 🤷

Reply:
I don't think the change was intentional. Probably it was just a bug that was not detected because we don't have a dashboard with ITKv4.

Reply:
Thanks @lassoan Do pull request (like this one) also appear at https://slicer.cdash.org/index.php?project=SlicerPreview ?

Reply:
Extensions need to set up their own CI to test not-yet-merged pull requests. There is a slicer docker image that is updated nightly that could be used for this, but I have not done it for any of my projects yet. It would be nice, but so far I've been just testing risky-looking PRs locally.

Reply:
Thanks @lassoan So do you consider this PR risky-looking? 😃